src/BuildingRegistry.Api.BackOffice/BackOfficeContext.cs

```diff
@@ -0,0 +1,82 @@
+namespace BuildingRegistry.Api.BackOffice
+{
+    using System;
+    using System.IO;
+    using Microsoft.EntityFrameworkCore;
+    using Microsoft.EntityFrameworkCore.Design;
+    using Microsoft.Extensions.Configuration;
+    using BuildingRegistry.Infrastructure;
+
+    public class BackOfficeContext : DbContext
+    {
+        public BackOfficeContext() { }
+
+        public BackOfficeContext(DbContextOptions<BackOfficeContext> options)
+            : base(options) { }
+
+        public DbSet<BuildingUnitBuildingRelationship> BuildingUnitBuildingRelationship { get; set; }
```

Comment:
`BuildingUnitBuilding` suffices. `BuildingUnitBuildings` for the table name / dbset